backend/tests/prisonerCsraHistory.test.js

```diff
@@ -0,0 +1,202 @@
+const prisonerCsraHistory = require('../controllers/prisonerProfile/prisonerCsraHistory')
```

Comment:
Is there a test that covers sorting?

Reply:
Yeah although there's no specific labelled test for it, it's covered under the `should render the correct template with the correct values` test. The mock data is ordered as **2011-06-02, 2014-11-10, 2013-11-10** but gets passed to the template in the correct descending order.